### ISSUE TITLE
The complete data is presented in DICe

### ISSUE THREAD

**Issue by chihyu722** [dicengine/dice#219]
Hello, 
When I finish the analysis and get the strain value of the segment (as shown in the figure below), can I get the data set of the complete line segment

![strain_xy](https://user-images.githubusercontent.com/87629687/126612560-4d77a633-b6db-4862-af23-e439ac9aa1af.PNG)

**Comment by dicengine** [comment]
Yes, this data should be in the file live_plot_line_fram_x.txt in the results folder.

**Comment by chihyu722** [comment]
Sorry, I would also like to ask: Can I get the full-field information in DICe (I did not find the relevant files in C:\Users\user\dice_working_dir), or where can I find the relevant files

**Comment by dicengine** [comment]
The full field results should be in the 'results' folder in the path you have above. There are DICe_Solution_[].txt files that have the output in comma separate values and a DICe_Solution.e file in exodus format which can be visualized in ParaView.
